logging-modules/log4j2-programmatic-configuration/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.baeldung.log4j2</groupId>
+	<artifactId>log4j2-programmatic-configuration</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<modules>
+		<module>simple-configuration</module>
+		<module>set-configuration-factory</module>
+		<module>simple-configurator</module>
+		<module>modifying-existing-config</module>
+		<module>simple-configuration-xml</module>
+		<module>custom-appender</module>
+		<module>custom-appenders</module>
+    <module>modify-xml-configuration</module>
```

Comment:
Let's fix the whitespace issues here. Would prefer spaces over tabs in the file.